app/controllers/tasks_controller.rb

```diff
@@ -0,0 +1,80 @@
+
+class TasksController < ApplicationController
+
+  def index
+    @tasks = Task.all.order(:id)
+  end
+
+  def show
+    task_id = params[:id].to_i
+    @task = Task.find_by(id: task_id)
+    if @task.nil?
+      redirect_to tasks_path
+    end
+  end
+
+  def new
+    @task = Task.new
+
+  end
+
+  def create
+    @task = Task.new(task_params) 
+    @task.update(completed: false)
```

Comment:
Update will actually modify _and_ commit/save to the database, so the `@task.save` on the line below is redundant. (`update` feels a lot like `create`, which is like `new` and `save`...) so the `save` is a little redundant here.